scripts/push_manifests.sh

```diff
@@ -20,7 +20,7 @@ git switch -c autopr-${RANDOM} origin/"${DEPLOYMENT_BRANCH}"
 # We assume the kustomize project already exists, this is not part of the PR
 cp ../"${WORKFLOW_ID}"/manifests/* kustomize/"${WORKFLOW_ID}"/base
 # Applying image kustomization
-cd kustomize/"${WORKFLOW_ID}"/overlays/prod || exit
```

Comment:
It's related to another change in the "config" repo structure that has never been adjusted in these script. do you prefer a dedicated PR? w/o this change the action would fail again

Reply:
no it's fine, I was just making sure it was not something you missed :)